index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <script src="https://unpkg.com/feather-icons"></script>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="main.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;400&display=swap" rel="stylesheet">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header class="flex pd-tp-bt">
+    <section class="welcome">
+      <h3>Welcome</h3>
+    </section>
+    <section class="menu">
+      <i class="fa fa-bars" aria-hidden="true"></i>
+    </section>
+  </header>
+  <main>
+    <article class="headline pd-tp-bt">
+      <h1>
+        <span class="clr-secondary">Hey there. I’m Mariane</span><span class="clr-primary block">I’m a software
+          developer</span>
+      </h1>
+      <section class="headline-info pd-tp-bt">
+        <p>
+          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you
+          see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </section>
+      <section class="social pd-tp-bt flex">
+        <i class="fa fa-twitter clr-secondary" aria-hidden="true"></i>
+        <i class="fa fa-linkedin clr-secondary" aria-hidden="true"></i>
+        <i class="fa fa-medium clr-secondary" aria-hidden="true"></i>
+        <i class="fa fa-github clr-secondary" aria-hidden="true"></i>
+        <i class="fa fa-angellist clr-secondary" aria-hidden="true"></i>
+      </section>
+
+      <img src="./images/pic1" alt="background image" class="abs pic1">
+      <img src="./images/pic2" alt="background image" class="abs pic2 wh-50">
+      <img src="./images/pic3" alt="background image" class="abs pic3" width="100" height="100">
+      <img src="./images/pic4" alt="background image" class="abs pic4">
+      <img src="./images/pic5" alt="background image" class="abs pic5 wh-50">
+      <img src="./images/pic6" alt="background image" class="abs pic6">
```

Comment:
- [ ] This section should use the font and colors (background and font) defined for the template